Tests/LeafErrorMiddlewareTests/LeafErrorMiddlewareTests.swift

```diff
@@ -169,6 +174,12 @@ class LeafErrorMiddlewareTests: XCTestCase {
         XCTAssertEqual(response.http.status, .forbidden)
         XCTAssertEqual(viewRenderer.leafPath, "serverError")
     }
+
+    func testThatRedirectIsNotCaught() throws {
+        let response = try app.getResponse(to: "/future303")
+        XCTAssertEqual(response.http.status, .seeOther)
+        XCTAssertEqual(response.http.body.string, "Test")
```

Comment:
I suspect this test will fail since we're not following the redirect through. You should be able to assert on `http.headers[.location]` though

Reply:
Strangely, it doesn't fail. I'm a little confused by the behaviour here, too - the status code is definitely 303 as I'd expect, but the location headers are empty. It looks like `ThrowingViewRenderer` explicitly throws away the original response in favour of a dummy response.

Reply:
Which is weird because it shouldn't even be hitting a view renderer 🤔